src/main.js

```diff
@@ -136,85 +130,85 @@ btnBackMain.addEventListener('click', switchPageMain)
 btnShowMain.addEventListener('click', switchPage)
 btnSave.addEventListener('click', savePoster)
 
-
-function savePoster(){
-    posterInstance = new Poster(
-    posterImg.src,
-    posterTitle.innerText,
-    posterQuote.innerText
-  )
-  savedPosters.push(posterInstance)
-};
-
-
 btnShowPoster.addEventListener('click', function() {
-  event.preventDefault()
-  posterTitle.innerText = makeTitle.value
-  posterImg.src = makeImage.value
-  posterQuote.innerText = makeQuote.value
-
-  titles.push(posterTitle.innerText)
-  images.push(posterImg.src)
-  quotes.push(posterQuote.innerText)
-  switchPage()
-
+    event.preventDefault()
+    posterTitle.innerText = makeTitle.value
+    posterImg.src = makeImage.value
+    posterQuote.innerText = makeQuote.value
+
+    titles.push(posterTitle.innerText)
+    images.push(posterImg.src)
+    quotes.push(posterQuote.innerText)
+    switchPage()
```

Comment:
I would recommend moving all of this into a named function outside of the event listener so It would look like 

`btnShowPoster.addEventListener('click', showPoster)

function showPoster() {
    posterTitle.innerText = makeTitle.value
    posterImg.src = makeImage.value
    posterQuote.innerText = makeQuote.value

    titles.push(posterTitle.innerText)
    images.push(posterImg.src)
    quotes.push(posterQuote.innerText)
    switchPage()
    }`
    
    This will clean up your code to make it a bit more readable